src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,79 @@
+package calculator
+
+import java.lang.IllegalArgumentException
+import java.util.regex.Pattern
+
+private const val FIRST_PREFIX =
+    """//"""
+
+private const val SECOND_PREFIX =
+    """\n"""
+
+private const val CUSTOM_SPLITTER_LOCATION = 2
+private const val CUSTOM_SPLITTER_LAST_LOCATION = 5
+// try {
+//        println("계산하기 위한 문자열을 입력해주세요.")
+//        val calculator = Calculator(readLine()!!)
+//        val result = calculator.execute()
+//        println("총합은 $result 입니다.")
+//    } catch (e: Exception) {
+//        println(e.message)
+//    }
+
+class Calculator(private val numbersInput: String) {
+
+    private val DEFAULT_SPLITTERS = listOf(",", ":")
+    private var isCustomSplitter: Boolean = false
+
+    fun execute(): Int {
+        getNumbers(numbersInput)
+        hasCustomSplitter()
+        hasOnlyValidString()
+        val numbers = parsing()
+        return calculate(numbers)
+    }
+
+    private fun calculate(numbers: List<Int>): Int {
+        return numbers.sum()
+    }
+
+    private fun getNumbers(numbersInput: String?): String? {
+        require(!numbersInput.isNullOrBlank()) { throw IllegalArgumentException("숫자를 입력해주세요.") }
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-lotto/pull/1#discussion_r463585709

```suggestion
        require(!numbersInput.isNullOrBlank()) { "숫자를 입력해주세요." }
```